specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2018-07-01/ResourceHealth.json

```diff
@@ -1327,6 +1403,10 @@
         "displayName": {
           "description": "The display name.",
           "type": "string"
+        },
+        "resourceTypes": {
+          "description": "list of resources",
+          "type": "array"
```

Comment:
if [type](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#parameterType) is "array", "items" is required.
Can you add "items" for "resourceTypes"?

Reply:
fixed